tests/unit/components/status/HeatMapColorBar.spec.js

```diff
@@ -0,0 +1,227 @@
+import { mount } from "@vue/test-utils";
+import { shallowMount } from "@vue/test-utils";
+import ComponentToTest from "@/components/status/HeatMapColorBar.vue";
+import { HeatMapColorBar as DistComponentToTest } from "@/dist/mantarray.common";
+
+import { createLocalVue } from "@vue/test-utils";
+
+let wrapper = null;
+
+const localVue = createLocalVue();
+
+describe("HeatMapColorBar.vue", () => {
+  afterEach(() => wrapper.destroy());
+  test("When mounting HeatMapColorBar from the build dist file, Then it loads successfully and upper_range text '100 μN' is rendered as defined", () => {
```

Comment:
The test title says "uN", but the test is using "uA" as the unit.  We don't want the title to say something inaccurate. However, I don't think it's necessary to say the specific value in the title--you can say something descriptive but not using specific values, like "Then it loads and renders the text for the top of the range with the number and units specified by the props"

also ,minor typo: `build`->`built`

Reply:
uN  is now ==> 'μN' and build is now  ==> 'built'